ui/src/translations/en.json

```
@@ -135,5 +136,10 @@
   "please_refer_to": "Please refer to",
   "import_solution_hint": "You should import a solution to prepare your environment",
   "redirecting": "Redirecting…",
-  "documentation": "Documentation"
+  "documentation": "Documentation",
+  "env_preparation": "Environment preparation",
+  "env_preparation_success": "Environment {envName} is being prepared.",
+  "env_preparation_failed": "Environemnt {envName} preparation failed.",
+  "preparing_environemnt": "preparing the environment",
+  "environment_creation_failed": "Environment {name} creation has failed."
```

For consistency with other keys:

```suggestion
  "env_creation_failed": "Environment {envName} creation has failed."
```